src/configuration.cpp

```diff
@@ -67,7 +67,7 @@ void irods::s3::set_resource(const std::string_view& resc)
 std::string irods::s3::get_resource()
 {
     if (!resource.has_value()) {
-        resource = g_config.value(nlohmann::json::json_pointer{"/s3_server/resource"}, std::string{});
+        resource = g_config.value(nlohmann::json::json_pointer{"/irods_client/resource"}, std::string{});
```

Comment:
Could change this to be like the other call. Notice the explicit template parameter in angle brackets.
```cpp
resource = g_config.value<std::string>(nlohmann::json::json_pointer{"/irods_client/resource"}, "");
```

Reply:
all the rest in that file are similar - i'll leave it for now.